src/runtime/ppx_deriving_runtime.cppo.ml

```diff
@@ -53,7 +53,7 @@ module Queue = Queue
 module Stack = Stack
 module Set = Set
 module Map = Map
-module Weak = Weak
+module Weak = Array
```

Comment:
I took a gander at the OCaml 4.08 changelog, am I missing the Weak changes?

(I'm trying to avoid modifying anything that exists upstream; if we *do* need to update anything, we should fork `ppx_deriving` directly, submit them, get them merged *there*, and then merge them downwards into `bs-deriving`)

Reply:
Bucklescript is not exporting Weak anymore :/